source/includes/_tipser-widget.md

```diff
@@ -199,27 +204,22 @@ The `onError` event handler is used with `useDefaultErrorHandler` config option.
     discountPriceIncVat: PriceModel;
     freeReturn: boolean;
 }
+
 ```
+Event handlers may be passed as a part of configuration. There is a number of events exposed for developer.
 
-The code example how to use event handlers:
+There is `eventsHandler` config option available. This is the place where you can add your own handlers for events.
+
+Currently we are supporting `onError` and `onAddToCart` event handlers.
+
+`onError` is being triggered when some error appear (i.e. when adding product to the cart failed) and `onAddToCart` event is being triggered when the product has been added to the cart.
+
+The handler for `onError` gets error argument (see code example). Error argument is object type with properties : `type`, `id`, `message`, `stack`. Currently all errors have `TipserElementError` and message contains the information for the user what went wrong. `Stack` is a typical error stack of js error.
```

Comment:
Not sure what you mean here: `Currently all errors have TipserElementError and message contains the information for the user what went wrong.` sth like => `Currently all errors contain TipserElementError element and message for the user what went wrong`?

Reply:
changed it a bit, I hope it will be more clear now, lemme know :)